src/h5web/visualizations/line/SpriteMaterial.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useLoader } from 'react-three-fiber';
+import { TextureLoader } from 'three';
+import { SpriteGlyph, Glyph } from './models';
+import circleURL from './sprites/circle.png';
+import crossURL from './sprites/cross.png';
+import squareURL from './sprites/square.png';
+
+const glyphURLs = {
+  [Glyph.Circle]: circleURL,
+  [Glyph.Cross]: crossURL,
+  [Glyph.Square]: squareURL,
+};
+
+interface Props {
+  color: string;
+  glyph: SpriteGlyph;
+}
+
+function SpriteMaterial(props: Props): JSX.Element {
+  const { color, glyph } = props;
+  const sprite = useLoader(TextureLoader, glyphURLs[glyph]);
+
+  return (
+    <pointsMaterial
+      attach="material"
+      map={sprite}
+      color={color}
+      size={6}
+      transparent
+    />
+  );
+}
+
+export default SpriteMaterial;
```

Comment:
Even though `Square` can be rendered by a simple `pointsMaterial`, I prefered to use a sprite to have a consistent behaviour for all `SpriteGlyph`.